src/frontends/tensorflow/src/op/linspace.cpp

```diff
@@ -0,0 +1,81 @@
+// Copyright (C) 2018-2022 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "op_table.hpp"
+#include "openvino/opsets/opset8.hpp"
+
+using namespace std;
+using namespace ov::opset8;
+
+namespace ov {
+namespace frontend {
+namespace tensorflow {
+namespace op {
+OutputVector translate_linspace_op(const NodeContext& node) {
+    /* LinSpace operation can be expressed in the following form:
+     * 1) Compute deltas by which each element will be increased in each slice
+     * through new dimension as delta = (stop - start) / (num - 1)
+     * 2) Generate a range of numbers by which times delta will be added to start to
+     * compute new elements in each slide as range = [0, 1, ..., num - 1]
+     * 3) Unsqueeze start and delta by new axis. And unsqueeze range by axes except given axis
+     * 4) Compute the result of the operation as result = start + delta * range
+     */
+    auto num_inputs = node.get_input_size();
+    auto start = node.get_input(0);
+    auto stop = node.get_input(1);
+    auto num = node.get_input(2);
+
+    TENSORFLOW_OP_VALIDATION(node, start.get_partial_shape().rank().is_static(), "Input rank must be static.");
+    int64_t rank = start.get_partial_shape().rank().get_length();
+
+    // retrieve axis from Constant node and compute a range of axes except given axis
+    // for unsqueezing start and delta tensors
+    std::vector<int64_t> axis;
+    std::vector<int64_t> except_axis_range;
+    if (num_inputs > 3 && rank > 0) {
+        get_const_input(node, 3, &axis);
+        TENSORFLOW_OP_VALIDATION(node, axis.size() == 1, "Axis must be a scalar for LinSpace operation.");
+        axis[0] = axis[0] >= 0 ? axis[0] : rank + 1 + axis[0];
```

Comment:
are you sure?
rank = 5
axis = -1 
with your formula 5 + 1 - 1 = 5, but it will be more then possible value (4)
I guess the correct variant: axis[0] = axis[0] >= 0 ? axis[0] : rank + axis[0];

Reply:
you seem to be correct. In case `-1`, I shouldn't do anything with it because `Unsqueeze` supports only [-R;R-1] range for axes

Reply:
As we discussed offline, currently it is correct. `-1` corresponds to the last dimension in output tensor and `[-R;R-1]` is a range of axes for output tensor for Unsqueeze operation in our opset